pyfar/dsp/dsp.py

```diff
@@ -1929,6 +1930,20 @@ def average(signal, mode='linear', caxis=None, weights=None, keepdims=False):
         If this is ``True``, the axes which are reduced during the averaging
         are kept as a dimension with size one. Otherwise, singular dimensions
         will be squeezed after averaging. The default is ``False``.
+    nan_policy: string, optional
+        Define how to handle NaNs in input signal.
+
+        ``'propagate'``
+           If the input signal includes NaNs, the corresponding averaged output
+           signal value will be NaN.
+        ``'omit'``
```

Comment:
I would suggest to add: "If a signal contains only NaN values in a specific dimensions, the output will be zero. For example if the second sample of a multi channel signal is allways NaN, the average will be zero at the second sample"

See this example:
```python
import pyfar as pf
import numpy as np

time = pf.TimeData(np.array([[1, np.nan, 1], [1, np.nan, 1]]), [1, 2, 3])
avg = pf.dsp.average(time, nan_policy="omit")
avg.time # yields [[1, 0, 1]]
```